### ISSUE TITLE
Speed Optimization on Intel VINO R1.2

### ISSUE THREAD

**Issue by mrinmayk** [opencv/dldt#119]
Hello,

On Intel VINO version R1.2 (2018.2.300), how much will a specific neural network like https://www.pyimagesearch.com/2018/05/07/multi-label-classification-with-keras/ be optimized to using the model optimizer in terms of speed (for processing a folder of images for e.g.)?

Does it come to 200%?

Regards

**Comment by dkurt** [comment]
@mrinmayk, x2 comparing to what?

**Comment by mrinmayk** [comment]
Comparing to running the same without VINO?

The networks to be optimized are given here- https://github.com/mahehu/TUT-live-age-estimator
